translations/translation_cz.xml

```diff
@@ -8,7 +8,7 @@
 		<text name="CP_unit_foot" 																		text="ft" />
 
 		<!-- User info text -->
-		<text name="CP_infoText"																		text="Courseplay(%s): Pro pomoc, chyby nebo poznámky o opravách se prosím navštivte: https://github.com/Courseplay/Courseplay_FS22. Český překlad: SniperKittenCZ, Chroja" />
+		<text name="CP_infoText"																		text="Courseplay(%s): Pro pomoc, chyby nebo poznámky o opravách se prosím navštivte: https://github.com/Courseplay/Courseplay_FS22. Český překlad: SniperKittenCZ (67,7%), Chroja (32,3%)" />
```

Comment:
Don't fight for the work, take it as teamwork!

Reply:
@Tensuko ok, i'm for the removal who is doing the translation, the @SniperKittenCZ has a problem with me signing there. @SniperKittenCZ probably not a team player. @SniperKittenCZ recently stated that I was appropriating his translation.